recipes/dipy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.11.0" %}
+
+package:
+  name: dipy
+  version: {{ version }}
+
+source:
+  fn: dipy-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/d/dipy/dipy-{{ version }}.tar.gz
+  md5: 7723e8994d31e910d47b7a0cf064a3d8
+
+build:
+    number: 0
+    script: python setup.py install
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
```

Comment:
We did end up having that for nibabel:

https://github.com/conda-forge/nibabel-feedstock/blob/master/recipe/meta.yaml#L15-L18

On Thu, May 12, 2016 at 9:07 AM, Gregory R. Lee <notifications@github.com>
wrote:

> In recipes/dipy/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/308#discussion_r63048289>
> :
>
> > +package:
> > +  name: dipy
> > +  version: {{ version }}
> > +
> > +source:
> > +  fn: dipy-{{ version }}.tar.gz
> > +  url: https://pypi.python.org/packages/source/d/dipy/dipy-{{ version }}.tar.gz
> > +  md5: 7723e8994d31e910d47b7a0cf064a3d8
> > +
> > +build:
> > +    number: 0
> > +    script: python setup.py install
> > +    features:
> > +        - vc9  # [win and py27]
> > +        - vc10  # [win and py34]
> > +        - vc14  # [win and py35]
>
> I may have been mistaken. These are not needed in order to compile with
> Cython on windows?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/308/files/96d561e50248b77046340e590026ecbca582cb38#r63048289>
>